src/coreclr/jit/emitarm64.cpp

```diff
@@ -2231,7 +2231,7 @@ void emitter::emitInsSanityCheck(instrDesc* id)
                 case INS_sve_st2w:
                 case INS_sve_st2d:
                 case INS_sve_st2q:
-                    assert(isValidSimm4_MultipleOf2(emitGetInsSC(id))); // iiii
+                    assert((isValidSimm_MultipleOf<4, 2>(emitGetInsSC(id)))); // iiii
```

Comment:
These extra parentheses are needed to avoid parsing issues from the comma in `<4, 2>` when expanding the `assert` macro.